test/test_owners.csv

```diff
@@ -360,6 +361,7 @@ Pods should support retrieving logs from the container over websockets,vishh,0
 "Port forwarding With a server that expects no client request should support a client that connects, sends no data, and disconnects",sttts,0
 PreStop should call prestop when killing a pod,ncdc,1
 PrivilegedPod should enable privileged commands,dchen1107,1
+PrivilegedPod should enable privileged commands,smarterclayton,0
```

Comment:
got told that you can ignore those test owner changes which are not related to the PR. CI will not complain. The script likes to transfer ownership far too often.